x-pack/plugins/secops/public/components/flyout/index.tsx

```diff
@@ -141,11 +134,9 @@ const mapStateToProps = (state: State, { timelineId }: OwnProps) => {
   return { show };
 };
 
-const bindActionsToDispatch = (dispatch: Dispatch) => ({
-  showTimeline: (id: string, show: boolean) => dispatch(timelineActions.showTimeline({ show, id })),
-});
-
 export const Flyout = connect(
   mapStateToProps,
-  bindActionsToDispatch
+  {
+    showTimeline: timelineActions.showTimeline,
```

Comment:
note: this follows the `mapDispatchToProps` pattern established in other areas and keeps the feature of being more true to the `flux` arch. by auto calling dispatch on actions as we want it to and less syntax noise.

I get to remove the `bindActionsToDispatch` above. If I were to have to keep that function I would have named it `mapDispatchToProps` to match the naming conventions